src/lazy/expanded/compiler.rs

```diff
@@ -491,8 +503,20 @@ impl TemplateCompiler {
             tdl_context,
             &mut compiled_body,
             /*is_literal=*/ false,
+            /*target_parameter=*/ None, /*; this is not a macro invocation arg.*/
             body,
         )?;
+        // Make sure there aren't any unexpected expressions following the body.
```

Comment:
Previously, expressions following the body would be ignored. Now this raises an error.